.github/workflows/go.yml

```diff
@@ -5,11 +5,11 @@ jobs:
     name: Test and Fmt
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/setup-go@v2
-        name: Setup go
+      - uses: actions/setup-go@v4
+        name: Setup Go
         with:
-          go-version: 1.14
-      - uses: actions/checkout@v2
+          go-version-file: "go.mod"
```

Comment:
If you use the go-version-file, the checkout step must be before it

Reply:
Fixed